src/main/kotlin/no/nav/yrkesskade/brevutsending/service/BrevService.kt

```diff
@@ -70,11 +67,7 @@ class BrevService(
             journalfoerendeEnhet = "9999", // erstatte med enhet fra token
```

Comment:
Burde disse to verdiene også blitt satt? Eller må det løses i en annen branch (har vi token?)

Reply:
Tenker jeg setter enhetsverdien i branch for distribuering. Men det er dette som er tanken bak at vi sender enhet i Brev-dataen over Kafka.
Så er jeg fortsatt litt usikker på hva eksternReferanseId bør være.